2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,45 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+const recipe1 = {
+  title: "Mole1",
+  Serves: 2,
+  Ingredients: ["cinnamon1", "cumin1", "cocoa1"],
+};
+const recipe2 = {
+  title: "Mole2",
+  Serves: 2,
+  Ingredients: ["cinnamon2", "cumin2", "cocoa2"],
+};
+const recipe3 = {
+  title: "Mole3",
+  Serves: 2,
+  Ingredients: ["cinnamon3", "cumin3", "cocoa3"],
+};
+const recipe4 = {
+  title: "Mole4",
+  Serves: 2,
+  Ingredients: ["cinnamon4", "cumin4", "cocoa4"],
+};
+const recipe5 = {
+  title: "Mole5",
+  Serves: 2,
+  Ingredients: ["cinnamon5", "cumin5", "cocoa5"],
+};
+
+const myFavouriteRecipes = [recipe1, recipe2, recipe3, recipe4, recipe5];
```

Comment:
Well done for solving this problem :) here I just have some suggestions.
You can either have one for loop  and console.log everything and another for loop just for showing the Ingredients items (inside of that for loop, they should be nested loop but you do not need to start the second for loop from the start) OR  (maybe this looks complicated but have a look at it and you can get some ideas :) ) 
`myFavouriteRecipes.map(eachRecipe => {console.log(eachRecipe.title);console.log(eachRecipe.Serves);console.log(eachRecipe.Ingredients.forEach(eachIngredients => console.log(eachIngredients)));})`

Reply:
thank you so much Ellietms, i will rewrite this part.